site/content/en/guides/consumer/function/_index.md

```diff
@@ -165,7 +168,6 @@ Here, rather than specifying `gcr.io/kpt-functions/label-namespace` using the
 [container runtime]: ../../producer/functions/container/
 [executables]: ../../producer/functions/exec/
 [starlark scripts]: ../../producer/functions/starlark/
-[`kpt cfg set`]: ../../../reference/cfg/set/
 [label-namespace]: https://github.com/GoogleContainerTools/kpt-functions-sdk/blob/master/ts/hello-world/src/label_namespace.ts
 [VCS]: https://en.wikipedia.org/wiki/Version_control
 [functions catalog]: catalog/
```

Comment:
Is this link still valid?

Reply:
The path itself is valid, but nothing on the page was referencing it anyway.

Reply:
It's a blank page and the Next Steps section references it. Let's remove it.

Reply:
Ah, this looked like it was highlighting the deleted change, but it sounds like this is referring to the broken `functions catalog` link. Done.